_data/developer.yml

```diff
@@ -266,6 +266,14 @@ websites:
       sms: Yes
       doc: https://www.vagrantup.com/docs/vagrant-cloud/users/authentication.html#two-factor-authentication
 
+    - name: IBM Cloud App ID
```

Comment:
IBM Cloud App ID is what I'd classify as a authentication provider. We no longer list [providers](https://twofactorauth.org/providers/).
Please remove IBM Cloud App ID from this pull request.

Reply:
Done and my apologies.